doc/book/api.md

```diff
@@ -0,0 +1,194 @@
+API
+===
+
+## Request Message
+
+`Zend\Diactoros\Request` implements `Psr\Http\Message\RequestInterface`, and is
```

Comment:
I prefer if line length match visible GH space. 80 characters limit is too few (I hope any text editor will reflow before to send to print).